themes/src/main/resources/theme/keycloak.v2/login/template.ftl

```diff
@@ -6,13 +6,33 @@
     <meta charset="utf-8">
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
     <meta name="robots" content="noindex, nofollow">
+    <meta name="color-scheme" content="light dark">
 
     <#if properties.meta?has_content>
         <#list properties.meta?split(' ') as meta>
             <meta name="${meta?split('==')[0]}" content="${meta?split('==')[1]}"/>
         </#list>
     </#if>
     <title>${msg("loginTitle",(realm.displayName!''))}</title>
+    <script type="module" async blocking="render">
```

Comment:
This script tag should be rendered based on whether dark mode is enabled or not. If has no purpose if dark mode support is not enabled.